docs/release_notes.md

```diff
@@ -49,7 +49,7 @@
   
 ### 📈 Improvements
 
--
+- [Orchestration] Supported new Data Masking config : `DPICustomEntity` with `replacement_strategy`.
```

Comment:
The release notes are meant for the customer to use the code, you should showcase the API that you added, not talk about internal details.
`DpiMasking#withRegex` is what the customer will use.